### ISSUE TITLE
Ignore children modules when formatting

### ISSUE THREAD

**Issue by LukasKalbertodt** [DamnWidget/anaconda_rust#15]
I'm a bit confused: why has no one noticed this yet? :P Or do I miss anything?

When auto-formatting, anaconda_rust does not pass `--skip-children`. Therefore when auto-formatting the parent module, the children's contents are copied into it as well. I think the fix is just to add said CLI flag...

**Comment by DamnWidget** [comment]
Right now that happens because the option is silently ignored by the anaconda's JsonServer (that is a bug), I did not fixed it yet because there is a lot going on with the `--skip-children` flag and `rustfmt`.

Long history short:
I want to get rid of the need of save a temporary file in order to run the `rustfmt` tool, to do that, I **must** use a `pipe` for the tool `stdin` and inject the file buffer trough it to `rustfmt`, the problem with that is that by default, `rustfmt` hardcode internally the write mode `Plain` when stdin is used and ignores any configuration (including `--skip-children`). In the other hand, `rustfmt` 0.5.0 adds the `--skip-children` option automatically when the `Plain` write mode is in use (that makes perfect sense as we don't care of the sub modules when using `Display` or `Plain`). 

I have some updates prepared but I didn't released them yet, in next versions we are gonna be able to call the Rust code in `rustfmt` directly form the Python code using FFI to solve the problem and in case that CFFI (python C Foreign Function Interface) could not be installed it will work properly with `rustfmt` 0.5.0

**TL;DR** best workaround for now is to locate a `rustfmt.toml` file in your project directory with the following contents:

``` toml
skip_children = true
```

**Comment by LukasKalbertodt** [comment]
Your workaround doesn't seem to work with my `rustfmt 0.5.0`... I added the hardcoded `--skip-children` flag locally now. Do you know when you could find time to deal with this? :) I plan to use this plugin with around 20 people for three weeks starting on Monday.

**Comment by DamnWidget** [comment]
Added commit, please, close if it fix your issue
